packages/react/src/components/PageTitleBar/PageTitleBar.jsx

```diff
@@ -372,9 +372,13 @@ const PageTitleBar = ({
                       ))
                     : null}
                   {headerMode === HEADER_MODES.DYNAMIC || headerMode === HEADER_MODES.CONDENSED ? (
-                    <span className="page-title-bar-breadcrumb-current" title={title}>
+                    <BreadcrumbItem
```

Comment:
Does this `BreadcrumbItem` support taking a title? I worry about test breaking if someone is using `getByTitle` or some other such combination.